lib/forest_liana/bootstrapper.rb

```diff
@@ -18,6 +18,14 @@ def initialize
         ForestLiana.auth_secret = ForestLiana.auth_key
       end
 
+      unless Rails.application.config.action_controller.perform_caching || Rails.env.test? || ForestLiana.forest_client_id
+        FOREST_LOGGER.error "You need to enable caching on your environment to use Forest Admin.\n" \
+          "For a development environment, run: `rails dev:cache`\n" \
+          "Or setup a static forest_client_id by following this part of the documentation:\n" \
+          "https://docs.forestadmin.com/documentation/how-tos/maintain/upgrade-notes-rails/upgrade-to-v6#setup-a-static-clientid"
+        exit(false)
```

Comment:
Actually we should not prevent the server to start in any case because of Forest Admin agent misconfiguration, as the server is also used for everything else.
The error message is good enough to alert the developer.
(+ the step is now clearly explained during the onboarding, so this error message should not be triggered a lot during the onboarding)